code.cpp

```diff
@@ -0,0 +1,116 @@
+#include <array>
+#include <iostream>
+#include <list>
+#include <unistd.h>
+
+class Table{
+    public:
+        static const int width = 32;
+        static const int height = 11;
+        static const int table_size =  width * height;
```

Comment:
Add also array with cells (better use std::vector instead of array and ask width and height from console)
Also add list with history as class member and make all the functions as class methods. 
Rename class to GameOfLife